engine/src/flutter/impeller/renderer/backend/vulkan/shader_library_vk.cc

```diff
@@ -48,16 +48,44 @@ static std::string VKShaderNameToShaderKeyName(const std::string& name,
   return stream.str();
 }
 
+namespace {
+bool SupportsFloat16(const std::weak_ptr<DeviceHolderVK>& device_holder) {
```

Comment:
I think you should change the test harness that is loading this shader to skip instead of changing the engine to ignore all shaders named "half".

Reply:
You need to do a runtime check to know if you can support it though.  That would require creating some canary subprocess to check the gpu capabilities during build time.

Reply:
Done